docs/tutorials/reply_bot.rst

```diff
@@ -210,17 +208,16 @@ to turn your Python script into a Python module, i.e., import it from another Py
 script or module. A common way to do that is to move the top-level code to a ``main``
 function:
 
-.. include:: ../examples/lmgtfy_bot.py
-    :code: python
-    :start-line: 8
-    :end-line: 16
+.. literalinclude:: ../examples/lmgtfy_bot.py
+    :language: python
+    :pyobject: main
 
 Finally, we need to call ``main`` only in the cases that this script is the one being
 executed:
 
-.. include:: ../examples/lmgtfy_bot.py
-    :code: python
-    :start-line: 35
+.. literalinclude:: ../examples/lmgtfy_bot.py
+    :language: python
+    :lines: 41-
```

Comment:
this should be 40, because it shows up like this
![image](https://user-images.githubusercontent.com/9664767/198824665-0d04b381-0712-4953-a3d1-25900db03cce.png)